From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lucius Hu <orctarorga@gmail.com>
Date: Tue, 4 Jun 2019 21:38:02 -0400
Subject: [PATCH] Correct path name in _ports

This is a patch for `zsh` package.

`./zsh/Completion/Unix/Type/_ports` file is a `zsh` completion function, which is installed to
`/usr/share/zsh/$ZSH_VERSION/functions/_path`. I corrected the path name to the file it's reading.

Replace `/etc/services` with
`/usr/share/defaults/etc/services`
---
 Completion/Unix/Type/_ports | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Completion/Unix/Type/_ports b/Completion/Unix/Type/_ports
index 8c8da53..7c26e7f 100644
--- a/Completion/Unix/Type/_ports
+++ b/Completion/Unix/Type/_ports
@@ -4,7 +4,7 @@ local expl ports
 
 if ! zstyle -a ":completion:${curcontext}:" ports ports; then
   (( $+_cache_ports )) ||
-      : ${(A)_cache_ports:=${${(M)${${(f)"$(</etc/services)"}:#\#*}#*/tcp}%%[ 	]*}}
+      : ${(A)_cache_ports:=${${(M)${${(f)"$(</usr/share/defaults/etc/services)"}:#\#*}#*/tcp}%%[ 	]*}}
 
   ports=( "$_cache_ports[@]" )
 fi
-- 
https://clearlinux.org

